.github/ISSUE_TEMPLATE/1.bug.yml

```diff
@@ -88,14 +88,17 @@ body:
 
   - type: dropdown
     attributes:
-      label: Do you use OpenAI GPT-3 or GPT-4?
+      label: What LLM Provider do you use?
       description: >
         If you are using AutoGPT with `SMART_LLM=gpt-3.5-turbo`, your problems may be caused by
         the [limitations](https://github.com/Significant-Gravitas/AutoGPT/issues?q=is%3Aissue+label%3A%22AI+model+limitation%22) of GPT-3.5.
       options:
-        - GPT-3.5
-        - GPT-4
-        - GPT-4(32k)
+        - GPT-3.5 (OpenAI)
+        - GPT-4 (OpenAI)
+        - GPT-4(32k) (OpenAI)
+        - GPT-4 (Azure)
+        - Anthropic
+        - Llamafile
```

Comment:
Should this be just a text field? There's lots of options and combinations possible, so a dropdown may be too limiting.

Reply:
IMO the model isn't super useful alone. Would be cool to just have them upload a config

Reply:
is that an option?? @kcze what would the appropriate location to ask for this from be?

Reply:
Sorry, I've missed this.
Yes, if they created the json configuration then they probably know where it is (directly in agent dir), otherwise the defaults are used.
However, currently model info isn't in json configuration, it's still in the env-based `AppConfig` (formerly legacy `Config`).